src/features/landing/components/Sections/ContributeSection.tsx

```
@@ -1,6 +1,7 @@
 import Link from 'next/link';
 import { SectionHeading } from '@/components/common/SectionHeading';
 import { Container } from '@/components/common/Container';
+import { Check, Map, PlusCircle } from 'lucide-react';
```

Biome (1.9.4)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)